app.js

```diff
@@ -136,17 +135,16 @@ if (Config.crashguard) {
 
 global.Sockets = require('./sockets');
 
-exports.listen = function (port, bindAddress, workerCount) {
-	Sockets.listen(port, bindAddress, workerCount);
+exports.listen = function (port) {
```

Comment:
Ok, I see what you mean. Would it still be a good idea to allow them to be passed from argv anyways?